examples/dmd/wave_equation.cpp

```diff
@@ -0,0 +1,621 @@
+ /*
+ * Copyright (c) 2013-2022, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+//
+//                       libROM MFEM Example: Wave_Equation (adapted from ex23.cpp)
+//
+// Compile with: make wave_equation 
+//
+// =================================================================================
+//
+// Sample runs and results for Time-Windowing DMD:
+//
+// Command 1:
+//  wave_equation -o 4 -tf 5 -nwinsamp 25 
```

Comment:
Does the example run in parallel?

Reply:
No, this is a serial time-windowing DMD implementation.

Reply:
In that case, all usage of MPI and myid can be removed.

Reply:
All usage of these parallel constructs has been removed.